README.md

```diff
@@ -1 +1,38 @@
-# Group_E_Simulation_Life_Cycle
\ No newline at end of file
+# Group_E_Simulation_Life_Cycle
+Françoise Blanc
+Roman Cardenas Rodriguez
+Ghida Dandan
+Aidan Fahlman
+Bruno St-Aubin
+
+## Description
+Need to Develop this Section
+
+
+## Software Requirements
+1. Users must be able to provide geospatial data as a source for the model generation workflow:
+	1. All geometric primitives (point, line and polygon)
+	2. Spatial network data  
+	3. In all common geospatial data formats (shapefile, geopackages, csv, geojson, etc.)
+2. Users must be able to provide attribute data in common formats (csv, excel, json) for the model generation process.
+3. Users must be able to provide a workflow file that contains the steps for automated model generation
+4. The system must automatically execute the workflow to convert geospatial data into a Cadmium model configuration file.
+	1. Map geospatial features onto simulation models from an existing library of parametric models
+	2. Establish neighborhoods or couplings through spatial analysis of the geospatial data
+	3. The file output must be a json file properly formatted according to Roman’s model generation process
+5. The system must support common spatial analysis functions to establish neighborhood and parameters:
+	1. Topological relationships (intersection, contains, within, touches, etc.)
+	2. Network distance analysis
+	3. Volume analysis
+	4. Spatial buffers
+	5. Voronoi diagrams
+	6. Geospatial statistics
+6. The system must support common SQL like function to determine model parameters
+7. The system must automatically convert the model configuration into a Cadmium model using an existing configuration generator.
+8. The system must compile the simulator for the model, execute the simulation and return the results.
+9. The system must convert the Cadmium results into a common specification for web based visualization through existing conversion web service
```

Comment:
Aren't points 7 to 9 just an extension of point 4?

Reply:
Maybe we can move these requirements to the top of the list and remove the nested list of point 4

Reply:
I'm not sure they are exactly extensions of point 4. They will all require separate pieces of code. I thought they could be all presented as separate features. 

Point 4 is only the generation of the json file. 
Point 7 is the generation of the cpp files
Point 8 is the compilation
Point 9 is the execution

Maybe these could all be like sub-tasks of a main task. For those familiar with Agile terminology, this would be equivalent to an Epic with multiple Tasks.

Reply:
Nice. Maybe for the first deliverable, we can provide (for instance) 3 points, and the second deliverable would add the remaining point and integrate all the points.

Reply:
This conversation has been addressed.